### PR TITLE
Retransmit data, not packets

### PR DESCRIPTION
**Pull request by ianswett**

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -496,7 +496,7 @@ loss_detection_alarm:
 : Multi-modal alarm used for loss detection.
 
 handshake_count:
-: The number of times the handshake packets have been
+: The number of times all unacknowledged handshake data has been
```
Comment (siyengar) [inline draft-ietf-quic-recovery.md]: "The number of times all unacknowledged handshake data has been transmitted"

without receiving an ack seems redundant :)

Comment (ianswett) [reply draft-ietf-quic-recovery.md]: It's consistent with tlp_count and rto_count because all get reset to 0 when an ack is received.

Comment (MikeBishop) [reply draft-ietf-quic-recovery.md]: I think the point is that "unacknowledged ... without receiving an ack" is either redundant (if it's unacknowledged, of course you didn't receive an ack) or ambiguous (without receiving an ack of *that* data, or without receiving *any* acks for anything?) depending how you read it.

Comment (ianswett) [reply draft-ietf-quic-recovery.md]: It's without receiving an ack for any new packets.  Does that deserve clarification in all 3 spots?

**Pull request merged**